src/MSBuild/MSBuild.csproj

```diff
@@ -216,16 +216,9 @@
     <Content Include="MSBuild.rsp" Pack="false">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <None Include="MSBuild\Microsoft.Build.CommonTypes.xsd">
-      <SubType>Designer</SubType>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="MSBuild\Microsoft.Build.Core.xsd">
-      <SubType>Designer</SubType>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="Microsoft.Build.xsd">
-      <SubType>Designer</SubType>
+    <XsdsForVS Include="**\*.xsd" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
```

Comment:
This hits extra xsds and sometimes multiple copies of one xsd, right? I don't think that's a problem, although it might hit performance slightly, but I wanted to check if I understood.

Reply:
I don't understand the question. Can you rephrase? Specifically what do you mean by "extra xsds" and by "multiple copies of one xsd"?

Reply:
I was thinking that if it's looking for anything.xsd, it would also find RuleSet.xsd, CustomDictionary.xsd, etc. and a different copy for each build configuration, but it occurred to me that that's all under the artifacts folder, so I think I was wrong.